hello-world-go-task/readme.md

```diff
@@ -0,0 +1,22 @@
+# Hello World Go
+
+Sample for Hello World task. Print "Hello World" when run the main.go
+
+## Getting started
+
+1. Fork this repository.
+2. Delpoy as a manual task in Choreo.
```

Comment:
Shall we describe this more?

something similar to:
"Please refer to the Choreo documentation under the [Develop a REST API](https://wso2.com/choreo/docs/develop-components/develop-services/develop-a-rest-api/#step-1-create-a-service-component-from-a-dockerfile) section to learn how to deploy the application.

You can select either Docker or Go as buildpacks. Fill as follow according to selected Buildpack."

Reply:
Fixed with https://github.com/wso2/choreo-samples/pull/15/commits/add09046e6b8c2d071d9af8ab6d9b1722a30038e